src/nl/hannahsten/texifyidea/settings/sdk/MiktexSdk.kt

```diff
@@ -25,10 +25,12 @@ class MiktexSdk : LatexSdk("MiKTeX SDK") {
 
     override fun suggestHomePaths(): MutableCollection<String> {
         val results = mutableSetOf<String>()
-        val path = "where pdflatex".runCommand()
-        if (path != null) {
-            val index = path.findLastAnyOf(setOf("miktex\\bin"))?.first ?: path.length - 1
-            results.add(path.substring(0, index))
+        val paths = "where pdflatex".runCommand()
+        if (paths != null) {
+            paths.split("\r\n").forEach { path ->
```

Comment:
would `System.lineSeparator()` be better?

Reply:
In theory, yes, but for MikTeX on Linux/Mac the `pdflatex` executable will not be present in that directory anyway - which is a separate problem I need to look into. When I do so, I might rename this one to WindowsMiktexSdk.